### PR TITLE
media-libs/gmmlib: version bump, added USE flag for user CFLAGS

### PR DESCRIPTION
**Pull request by Karlson2k**
* gmmlib updated to the latest upstream version 2.4.1
* Added patch and USE flag to disable override of user CFLAGS in cmake configuration. Now CFLAGS like -march=native are honoured.

### CODE REVIEW

File: media-libs/gmmlib/gmmlib-20.4.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+PATCHES=(
+	"${FILESDIR}"/${PN}-20.2.2_conditional_testing.patch
+	"${FILESDIR}"/${PN}-20.4.1_user_cflags.patch
+)
+
+multilib_src_configure() {
+	local mycmakeargs=(
+		-DBUILD_TYPE=Release
+		-DBUILD_TESTING=$(usex test)
+		-DOVERRIDE_COMPILER_FLAGS=$(usex !user-cflags)
```
Comment (thesamesam) [inline media-libs/gmmlib/gmmlib-20.4.1.ebuild:46]: custom-flags ;)

File: media-libs/gmmlib/metadata.xml [hunk 2]
```diff
@@ -1,7 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-<maintainer type="project">
-	<email>media-video@gentoo.org</email>
-</maintainer>
+  <maintainer type="project">
```
Comment (thesamesam) [inline media-libs/gmmlib/metadata.xml:4]: Add yourself, if you fancy!

File: media-libs/gmmlib/gmmlib-20.4.1.ebuild (hunk 1 above)
Comment (Karlson2k) [reply media-libs/gmmlib/gmmlib-20.4.1.ebuild:46]: I'm working on it already. :)

File: media-libs/gmmlib/metadata.xml (hunk 2 above)
Comment (Karlson2k) [reply media-libs/gmmlib/metadata.xml:4]: Already done. I will update PR soon.

**Comment by Karlson2k** [comment]
@thesamesam Updated.

**Comment by Karlson2k** [comment]
@thesamesam PR was updated as discussed.

**Comment by Karlson2k** [comment]
> But easily fixed.
> 
> I have exams this week! But I think it is a quick one, so I'll try do it now ;)

@thesamesam Sorry, missed this one.

Thanks for merging.

And good luck with your exams!

**Pull request closed without merging**